### PR TITLE
Update baggers.js

### PR DESCRIPTION
**Pull request by harrich**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13992,6 +13992,39 @@ var data = {
         "cities": [
             "Toulouse"
         ]
+    },
+    {
+      "since": "2017-01-01",
+      "name": "Harrich S",
```
Comment (dadoonet) [inline baggers.js]: Is `S` your real name? Better if you provide your full name (first+last)

File: baggers.js [hunk 2]
```diff
@@ -13992,6 +13992,39 @@ var data = {
...
+            "rebase",
+            "push"
+          ],
+          "lang": [
+            "fr"
+          ]
+        }
+      ],
+      "cities": [
+        "Nantes"
```
Comment (dadoonet) [inline baggers.js:14023]: You wrote in `location` that you are in both Paris and Nantes.
Don't you want to also appear in the list of Paris baggers?

**Review by dadoonet — COMMENTED** [review]
Left some comments. Can you check them please?

File: baggers.js (hunk 2 above)
Comment (harrich) [reply baggers.js:14023]: Fix.

File: baggers.js (hunk 1 above)
Comment (harrich) [reply baggers.js]: It's only my first letter. I prefer to leave it like that.

Comment (dadoonet) [reply baggers.js]: Well. It goes against what we wrote in our docs: https://github.com/brownbaglunch/bblfr_data#speakers-dataset

> `name`: your full name "First Last"

Is Harrich your first name or last name?

Comment (harrich) [reply baggers.js]: It's my first name.

Comment (dadoonet) [reply baggers.js]: So how people will know who they are inviting?

**Comment by dadoonet** [comment]
@brownbaglunch/owners Can you comment and give your opinion on https://github.com/brownbaglunch/bblfr_data/pull/164#discussion_r92999669 please?

**Comment by nrichand** [comment]
Hi @harrich glad you'd like to join us.
I agree with @dadoonet comment, I'm not a huge fan that your name is hidden, how will people know who your are and want to invite you? Can you please fill it?

Thanks,
Nathaniel

**Comment by harrich** [comment]
Done.

**Review by dadoonet — APPROVED** [review]

**Comment by dadoonet** [comment]
Thanks and welcome!

**Pull request merged**